cmd/bee/cmd/start.go

```diff
@@ -320,7 +321,14 @@ func (c *command) configureSigner(cmd *cobra.Command, logger logging.Logger) (co
 			return nil, err
 		}
 
-		signer, err = clef.NewSigner(externalSigner, clefRPC, crypto.Recover)
+		wantedAddress := c.config.GetString(optionNameClefSignerAddress)
+		var overlayEthAddress *common.Address = nil
+		if wantedAddress != "" {
+			ethAddress := common.HexToAddress(wantedAddress)
+			overlayEthAddress = &ethAddress
+		}
+
```

Comment:
Make a comment that if wantedAddress == "", we use the first-available clef account or generate a new one

Reply:
> Or generate a new one 

? Does this happen? See comment below

Reply:
done. and no, does not happen.